Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -72,20 +72,25 @@ public function flushNodeAggregate(
         $tagsToFlush[ContentCache::TAG_EVERYTHING] = 'which were tagged with "Everything".';
 
         $tagsToFlush = array_merge(
-            $this->collectTagsForChangeOnNodeAggregate($contentRepository, $workspaceName, $nodeAggregateId),
+            $this->collectTagsForChangeOnNodeAggregate($contentRepository, $workspaceName, $nodeAggregateId, $workspaceName),
```

Comment:
the workspace name appears twice in the list of parameters? is that intended?

Reply:
OK, I got it – if it's set, we flush only tags with the workspace otherwise we flush all..
This is pretty implicit that way. what do you think about some boolean `respectWorkspace` flag?

Reply:
I guess the question is if we could see a scenario where this second WorkspaceName is not the same as the first, but I guess that makes no sense given how the method is used. So yeah, I think with a bool would be fine.

Reply:
We already changed that. 😉